packages/atomic/src/components/facets-v1/atomic-facet/atomic-facet.tsx

```diff
@@ -164,6 +195,27 @@ export class AtomicFacet implements InitializableComponent {
     return this.facetState.values.map((value) => this.renderValue(value));
   }
 
+  private renderSearchResults() {
+    return this.facetState.facetSearch.values.map((value) =>
+      this.renderValue({
```

Comment:
We will reuse the functional component to have the same values visually, but we'll have a different handler for selection.